src/cpu/amd/pi/spi.c

```diff
@@ -17,7 +17,8 @@
 #include <spi-generic.h>
 #include <spi_flash.h>
 
-#include "s3_resume.h"
+// #include "s3_resume.h"
```

Comment:
@mek-x if it is not needed why not to remove it ?

Reply:
@pietrushnic it's left for future reference